.gitignore

```diff
@@ -286,3 +286,6 @@ __pycache__/
 *.btm.cs
 *.odx.cs
 *.xsd.cs
+
+
+!/Server/Sesame.Web/Properties/launchSettings.json
```

Comment:
this is checked in as part of the repo. Why is it ignored and checked in?

Reply:
ah I see - there's a bang at the start, meaning that the launchSettings.json file is explicitly NOT ignored.